pyigm/clustering/survey_2d2pcf.py

```diff
@@ -89,7 +95,7 @@ def addField(self, new_field, SEP_TOL=0.5*u.deg):
         fcoord = SkyCoord(ra=[field.CRA for field in self.fields],
                           dec=[field.CDEC for field in self.fields], unit='deg')
         if np.any(new_field.coord.separation(fcoord) < SEP_TOL):
-            raise IOError('New field is within 10" of an already input field!  We assume a mistake was made')
+            raise IOError('New field is within {} of an already input field!  We assume a mistake was made'.format(SEP_TOL))
```

Comment:
Please add to the error message: "... because the fields are supposed to be independent."

Reply:
added